### PR TITLE
Prompt func

### PR DESCRIPTION
**Pull request by Arsh-ANiket**
I have added a new file "finalprompt.py" with a function to generate a propmt.

### CODE REVIEW

**Review by Ashish-jukaria — COMMENTED** [review]
where are db changes .... ?

File: finalprompt.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+def prompt(category, plan_type, plan_timing, location, number_of_people,budget):
+
+    str=f"You are a {category} planner, guiding me through the process of planning a {category}.You should provide consistent and reliable advice and suggestions every time I make a request.\n\n I have {plan_type} marriage going on.List down the events for a basic one having budget of {budget}.Assign each event a key and also plan the best optimized budget for each events.give me the response in json having following format"
+    print(str)
+
+# store every record for passing it to db
+category = input("Enter the category of event: ")
```
Comment (Ashish-jukaria) [inline finalprompt.py:7]: remove input fields....
Put this string in prompts table.
Make a service called promptservice.py and write down the function there.

**Review by Ashish-jukaria — COMMENTED** [review]
Also remove this prompt scripts directory....its not needed.

File: finalprompt.py (hunk 1 above)
Comment (Arsh-ANiket) [reply finalprompt.py:7]: i have removed it from a seperate directory,
updated the function with no input fields such that the table data will be taken up and formatted

**Pull request closed without merging**